packages/canary-features/addon/index.js

```diff
@@ -1,17 +1,10 @@
 /* globals EmberDataENV */
 
 import { assign } from '@ember/polyfills';
+import { DEFAULT_FEATURES } from './default-features';
```

Comment:
If we migrate to a default export over in `./default-features`:

```suggestion
import DEFAULT_FEATURES from './default-features';
```